packages/sdk/src/utils.ts

```diff
@@ -31,7 +31,7 @@ export * from "./utils/substrate"
 
 export const DEFAULT_POLL_INTERVAL = 5_000
 export const ADDRESS_ZERO = "0x0000000000000000000000000000000000000000" as HexString
-export const MOCK_ADDRESS = "0x1234567890123456789012345678901234567890" as HexString
+export const MOCK_ADDRESS = "0xc8cb6c49f90d7024b5d4fd2ed5491dbffb80ca13" as HexString
```

Comment:
Had to update the mock address to a random user address because 0x123456789..... seems to be blacklisted for some stables.